packages/graphql_server/src/updateProject.ts

```diff
@@ -44,17 +49,33 @@ const updateAllProjectInfo = async (
   if (!(await repoIsAlreadyInDB(repoName, owner))) {
     return
   }
+  await updateProjectGithubStats(repoName, owner)
   await updateProjectELI5(repoName, owner)
   await updateProjectFounders(repoName, owner)
-  await updateProjectGithubStats(repoName, owner)
   await updateProjectLinkedInData(owner)
   await updateProjectSentiment(repoName, owner)
   await updateProjectStarHistory(repoName, owner)
+  await updateProjectForkHistory(repoName, owner)
```

Comment:
this is wrong

Reply:
There are design flaws absolutely. But what we do here is updating the info that we get. If let's say we don't get the hackernews sentiment of a project we would still want to add the project to the db with everything else